# Final Practice Problems

## Problem 1


In [13]:
xbar = 82.17
sigma = 2.2
mu = 81
alpha = 0.05

z = (xbar - mu) / (sigma / sqrt(12))
paste0("Z-score: ", round(z, 4))
z_critical = qnorm(1 - alpha/2)
paste0("Critical Z-value: ", round(z_critical, 4))

if (z > z_critical) {
  result = "Reject the null hypothesis"
} else {
  result = "Fail to reject the null hypothesis"
}
result


[1] "Z-score: 1.8423"

[1] "Critical Z-value: 1.96"

[1] "Fail to reject the null hypothesis"

## Problem 3

In [14]:
# Observed counts
obs <- c(292, 190, 18)

# Hypothesized probabilities: 60%, 35%, 5%
p0 <- c(0.60, 0.35, 0.05)

# Chi-square goodness-of-fit test
gof_test <- chisq.test(x = obs, p = p0)

gof_test$statistic   # test statistic
gof_test$p.value     # P-value


X-squared 
 3.459048

[1] 0.1773689

In [15]:
n  <- 500
exp <- n * p0

chi_sq <- sum((obs - exp)^2 / exp)
p_val  <- 1 - pchisq(chi_sq, df = length(obs) - 1)

chi_sq   # ≈ 3.46
p_val    # ≈ 0.18


[1] 3.459048

[1] 0.1773689

## Problem 4

In [19]:
n    = 15
xbar = 3.87
s = 1.45
mu0 = 4.69

t_stat = (xbar - mu0) / (s / sqrt(n))
crit = qt(0.10, df = n - 1)

paste0("t_stat: ", round(t_stat, 4))
paste0("crit: ", round(crit, 4))

if (t_stat < crit) {
  result = "Reject the null hypothesis"
} else {
  result = "Fail to reject the null hypothesis"
}
result

[1] "t_stat: -2.1902"

[1] "crit: -1.345"

[1] "Reject the null hypothesis"

## Problem 6: Two-Proportion Test

**Problem Setup:**
We want to test whether there's a significant difference between two population proportions. We have:
- Sample 1: 167 successes out of 200 trials
- Sample 2: 178 successes out of 200 trials

**Hypotheses:**
- $H_0: p_1 = p_2$ (no difference between proportions)
- $H_1: p_1 < p_2$ (proportion 1 is less than proportion 2) - this is a left-tailed test

**Solution Approach:**
1. Calculate sample proportions $\hat{p}_1$ and $\hat{p}_2$
2. Calculate pooled proportion $\hat{p}$ (assuming $H_0$ is true)
3. Calculate standard error using the pooled proportion
4. Compute the test statistic $z = \frac{\hat{p}_1 - \hat{p}_2}{SE}$
5. Find the p-value for the left-tailed test
6. Make a decision based on the significance level

In [23]:
p1 = 167
p2 = 178

p1.hat = p1 / 200
p2.hat = p2 / 200
p.pool = (p1 + p2) / 400

se = sqrt(p.pool * (1 - p.pool) * (1/200 + 1/200))

z = (p1.hat - p2.hat) / se
paste0("Z-score: ", round(z, 4))

p_value = round(pnorm(z), 3)
paste0("P-value: ", p_value)

z_critical = qnorm(0.95)
paste0("Critical Z-value: ", round(z_critical, 4))

if (z > z_critical) {
  result = "Reject the null hypothesis"
} else {
  result = "Fail to reject the null hypothesis"
}
result


[1] "Z-score: -1.5971"

[1] "P-value: 0.055"

[1] "Critical Z-value: 1.6449"

[1] "Fail to reject the null hypothesis"

We test

$$
H_0:p_1=p_2 \quad \text{vs} \quad H_1:p_1<p_2
$$

From the samples:

$$
\hat p_1 = \frac{167}{200} = 0.835,\qquad
\hat p_2 = \frac{178}{200} = 0.89
$$

Pooled proportion under $H_0$:

$$
\hat p = \frac{167+178}{200+200} = \frac{345}{400} = 0.8625
$$

Standard error:

$$
SE = \sqrt{\hat p(1-\hat p)\left(\frac1{n_1}+\frac1{n_2}\right)}
= \sqrt{0.8625(1-0.8625)\left(\frac1{200}+\frac1{200}\right)}
\approx 0.0344
$$

Test statistic:

$$
z = \frac{\hat p_1-\hat p_2}{SE}
= \frac{0.835-0.89}{0.0344}
\approx -1.60
$$

P-value for lower-tailed test:

$$
p = P(Z \le -1.60) \approx 0.055
$$

